test/library/standard/DataFrames/psahabu/HelloDataFrame.good

```diff
@@ -1,20 +1,20 @@
-         columnThree   columnTwo   columnOne   
-rowOne             1        10.0           a   
-rowTwo          None        20.0        None   
-rowThree           3        30.0           c   
-rowFour         None        40.0        None   
-rowFive            5        50.0           e   
+         columnOne   columnThree   columnTwo   
+rowOne           1          10.0           a   
+rowTwo        None          20.0        None   
+rowThree         3          30.0           c   
+rowFour       None          40.0        None   
+rowFive          5          50.0           e   
 
-rowOne      1
-rowTwo      None
-rowThree    3
-rowFour     None
-rowFive     5
-dtype: int(64)
+rowOne      10.0
+rowTwo      20.0
+rowThree    30.0
+rowFour     40.0
+rowFive     50.0
+dtype: real(64)
```

Comment:
Is this supposed to have changed? It looks like the row values are totally different and a different dtype?

Reply:
Taking a quick look at the test, it appears the new values are correct, it should be the real column. I will come back to this later today; it is concerning that this has changed... My initial thought is something with the DataFrame implementation that is only in the `test/` directory and not very well tested, but will need to explore that more.

Reply:
It seems what is going on here is that in the `DataFrame` implementation, the column names (`labels`) are stored as an associative domain and the actual columns are stored as an array. This means that, while you might initialize your `DataFrame` with parallel lists, they are not guaranteed to remain parallel since arrays are ordered and associative domains are not. This is why the diff of this file changes the column names in the printing of the whole data frame, even though the data stays the same.

Seeing that, it seems to be an implementation detail of this DataFrame and I'm not sure that I want to update this DataFrame code to match my interpretation of how it should work, having no context for why this code exists in the `test/` directory. 

With this information, I am inclined to just disregard this difference of ordering here and possibly open an issue about that DataFrame code, even though it only exists in the `test/` directory, it seems we still don't want code that is being tested to have errors.

Reply:
Yeah good catch it does look like it should be printing columnThree which is not what was happening before...

Reply:
fixed in https://github.com/chapel-lang/chapel/pull/18870/commits/bfd448024e2d1dd53f9a1b0bff136d6db918d4b8